src/components/Sum.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a, b }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
```suggestion
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
```
так буде більш читабельно, багато різних символів, очі розбігаються))